### ISSUE TITLE
Namespace should only be created if it does not already exist

### ISSUE THREAD

**Issue by hyder** [oracle-terraform-modules/terraform-oci-oke#387]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

The OCIR secret functionality allows the creation of the secret in a namespace of the user's choice. However, since the default value of the namespace for the ocir secret is "default" and the "default" namespace already exists in OKE, it should not create try to create it again. 

### Terraform Configuration Files

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. 
# Please remove any sensitive information from configuration files before sharing them. 
```

### Steps to Reproduce

1. `terraform apply`

**Comment by karthicgit** [comment]
The way k8s manifest is used we don't need the check for existing namespace. It wont throw error if the namespace exists.

**Comment by hyder** [comment]
It did

**Comment by denismakogon** [comment]
the way it was adjusted recently - it wouldn't raise an issue because it goes through `kubectl apply` that wouldn't through an error if a resource exists.

**Comment by hyder** [comment]
fixed in #394
